featureflags.go

```diff
@@ -274,3 +255,65 @@ func (poller *FeatureFlagsPoller) ForceReload() {
 func (poller *FeatureFlagsPoller) shutdownPoller() {
 	poller.shutdown <- true
 }
+
+func (poller *FeatureFlagsPoller) getFeatureFlagValue(featureFlag FeatureFlag, key string, distinctId string) (interface{}, error) {
```

Comment:
`getFeatureFlagValue` is more accurate but since we've already implemented the function as `getFeatureFlag` in our js libraries etc how about keeping it as `getFeatureFlag` for consistency?

Reply:
nvm you did leave it as getFeatureFlag in the client interface it appears, ignore meee 😃